pom.xml

```diff
@@ -32,8 +32,8 @@
         <!-- Versioning -->
         <version.arquillian_core>1.1.11.Final</version.arquillian_core>
         
-        <weld.api.version>2.3.SP2</weld.api.version>
-        <weld.core.version>2.3.5.Final</weld.core.version>
+        <weld.api.version>2.4.CR1</weld.api.version>
+        <weld.core.version>2.4.0.CR1</weld.core.version>
```

Comment:
You are correct. It wasn't implemented. Are there any objections to merge this PR in its current shape?